src/lazy/binary/raw/v1_1/reader.rs

```diff
@@ -283,15 +282,6 @@ mod tests {
             // Symbol: 'variable length encoding'
             0xF9, 0x31, 0x76, 0x61, 0x72, 0x69, 0x61, 0x62, 0x6C, 0x65, 0x20, 0x6C, 0x65, 0x6E,
             0x67, 0x74, 0x68, 0x20, 0x65, 0x6E, 0x63, 0x6f, 0x64, 0x69, 0x6E, 0x67,
-
-            // Symbol ID: 1
-            // 0x51, 0x01,
-
-            // // Symbol ID: 257
-            // 0x52, 0x01, 0x00,
-
-            // // Symbol ID: 65,793
-            // 0x50, 0x00, 0x00, 0x00,
```

Comment:
We should add these back in (with the appropriate opcodes) now that you've updated the symbol ID handling.